### PR TITLE
[es] Add translations for Laravel Spark

### PR DESCRIPTION
**Pull request by luisprmat**

### CODE REVIEW

File: locales/es/es.json [hunk 1]
```diff
@@ -272,20 +273,20 @@
...
+    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Si necesita agregar información de contacto específica o de impuestos a sus recibos, como su nombre comercial completo, número de identificación VAT o dirección de registro, puede agregarlo aquí.",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Si tiene problemas para hacer clic en el botón \":actionText\", copie y pegue la siguiente URL \nen su navegador web:",
     "Increase": "Incrementar",
     "India": "India",
     "Indonesia": "Indonesia",
     "Invalid signature.": "Firma no válida.",
-    "Iran, Islamic Republic of": "Iran, Islamic Republic of",
+    "Iran, Islamic Republic of": "Irán, República Islámica de",
```
Comment (caouecs) [inline locales/es/es.json:282]: you have two translations for the same country ?

File: locales/es/es.json [hunk 2]
```diff
@@ -300,11 +301,11 @@
     "Kiribati": "Kiribati",
     "Korea": "Corea del Sur",
     "Korea, Democratic People's Republic of": "Corea del Norte",
-    "Korea, Republic of": "Korea, Republic of",
+    "Korea, Republic of": "Corea, República de",
```
Comment (caouecs) [inline locales/es/es.json:304]: Republic of Korea is South Korea

File: locales/es/es.json [hunk 3]
```diff
@@ -328,7 +329,7 @@
     "Luxembourg": "Luxemburgo",
     "Macao": "Macao",
     "Macedonia": "Macedonia",
-    "Macedonia, the former Yugoslav Republic of": "Macedonia, the former Yugoslav Republic of",
+    "Macedonia, the former Yugoslav Republic of": "Macedonia, ex República Yugoslava de",
```
Comment (caouecs) [inline locales/es/es.json:332]: you do two translations ?

File: locales/es/es.json [hunk 4]
```diff
@@ -532,16 +533,16 @@
     "Sorry, your session has expired.": "Lo siento, su sesión ha caducado.",
     "South Africa": "Sudáfrica",
     "South Georgia And Sandwich Isl.": "Islas Georgias del Sur y Sandwich del Sur",
-    "South Georgia and the South Sandwich Islands": "South Georgia and the South Sandwich Islands",
+    "South Georgia and the South Sandwich Islands": "Georgia del sur y las islas Sandwich del sur",
```
Comment (caouecs) [inline locales/es/es.json:536]: Islas

File: locales/es/es.json [hunk 5]
```diff
@@ -551,10 +552,10 @@
     "Switzerland": "Suiza",
     "Syrian Arab Republic": "Siria",
     "Taiwan": "Taiwán",
-    "Taiwan, Province of China": "Taiwan, Province of China",
+    "Taiwan, Province of China": "Taiwan, provincia de China",
```
Comment (caouecs) [inline locales/es/es.json:555]: Taiwán ?

File: locales/es/es.json [hunk 6]
```diff
@@ -551,10 +552,10 @@
...
     "Tajikistan": "Tayikistán",
     "Tanzania": "Tanzania",
-    "Tanzania, United Republic of": "Tanzania, United Republic of",
+    "Tanzania, United Republic of": "Tanzania, República Unida de",
```
Comment (caouecs) [inline locales/es/es.json:558]: it's just Tanzania, no ?

File: locales/es/es.json [hunk 7]
```diff
@@ -644,67 +645,67 @@
...
     "Use a recovery code": "Use un código de recuperación",
     "Use an authentication code": "Use un código de autenticación",
     "Uzbekistan": "Uzbekistan",
     "Value": "Valor",
     "Vanuatu": "Vanuatu",
-    "VAT Number": "VAT Number",
+    "VAT Number": "Número VAT",
     "Venezuela": "Venezuela",
-    "Venezuela, Bolivarian Republic of": "Venezuela, Bolivarian Republic of",
+    "Venezuela, Bolivarian Republic of": "Venezuela, República Bolivariana de",
```
Comment (caouecs) [inline locales/es/es.json:667]: Venezuela ?

File: locales/es/es.json [hunk 8]
```diff
@@ -644,67 +645,67 @@
...
+    "Your card was declined. Please contact your card issuer for more information.": "Su tarjeta fue rechazada. Comuníquese con el emisor de su tarjeta para obtener más información.",
+    "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Su método de pago actual es una tarjeta de crédito que termina en :lastFour que vence el :expiration.",
     "Your email address is not verified.": "Su dirección de correo electrónico no está verificada.",
-    "Your registered VAT Number is :vatNumber.": "Your registered VAT Number is :vatNumber.",
+    "Your registered VAT Number is :vatNumber.": "Su número VAT registrado es :vatNumber.",
+    "Your :invoiceName invoice is now available!": "¡Su factura :invoiceName ya está disponible!",
     "Zambia": "Zambia",
     "Zimbabwe": "Zimbabwe",
-    "Zip \/ Postal Code": "Zip \/ Postal Code"
+    "Zip / Postal Code": "Zip / Código postal"
```
Comment (caouecs) [inline locales/es/es.json:710]: need "\\"

**Review by caouecs — CHANGES_REQUESTED** [review]
Problem with country names.

File: locales/es/es.json (hunk 5 above)
Comment (luisprmat) [reply locales/es/es.json:555]: [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) is dufplicate from `source/en.json` not `es.json`

File: locales/es/es.json (hunk 1 above)
Comment (luisprmat) [reply locales/es/es.json:282]: @caouecs, i didn't write this duplicate keys in file `source/en.json` but you. I only translate. For example
[https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L282](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L282) Lines 281 and 282 are the same contry. The same goes for the other cases.
If you don't want repeated countries it's necesary modify `source/en.json` in several parts.
- [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L330](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L330) 330 and 331
- [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) 553 and 554
- etc ...

File: locales/es/es.json (hunk 2 above)
Comment (luisprmat) [reply locales/es/es.json:304]: [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) is duplicated in `source/en.json`

File: locales/es/es.json (hunk 3 above)
Comment (luisprmat) [reply locales/es/es.json:332]: I only translate `locales/es/es.json` based in `source/en.json` and in this one the countries was repeated.

**Comment by luisprmat** [comment]
You decide which keys we delete from `en.json` and we also arrange `es.json` so that they remain the same. 
[https://github.com/Laravel-Lang/lang/discussions/1702#discussioncomment-662971]

File: locales/es/es.json (hunk 8 above)
Comment (luisprmat) [reply locales/es/es.json:710]: Problems in [`source/en.json`](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L709). There is the same problem

**Comment by luisprmat** [comment]
**Conclusion**: Please help me fix the `source/en.json` with the correct keys (which I don't really know what they are) and I will gladly fix mine based on that.

File: locales/es/es.json (hunk 7 above)
Comment (luisprmat) [reply locales/es/es.json:667]: [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L665](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L665)

**Comment by caouecs** [comment]
In en.json, we add Laravel Spark tags, and we have duplication of country names

File: locales/es/es.json (hunk 8 above)
Comment (caouecs) [reply locales/es/es.json:710]: I will fix source/en.json

**Comment by caouecs** [comment]
Grasias

**Pull request merged**